wiki/Game_Modifiers/en.md

```diff
@@ -1,297 +1,333 @@
+In [osu!taiko](/wiki/Game_Modes/osu!taiko) mode, the notes fade out about halfway through the screen, requiring that the player remembers their timing and color. However, sliders and spinners still cross the full timeline and do not fade out, except that the spinner has no approach circle to indicate when it times out. On maps with high overall difficulty, Hidden is used by experienced players to increase score instead of Hard Rock, because Hard Rock sometimes makes the timing window much too small.
+
+In [osu!catch](/wiki/Game_Modes/osu!catch) mode, Hidden causes notes to fade out about halfway down the screen. This is the most commonly used mod in Catch the Beat, since the other difficulty increasing mods are very difficult. They either greatly increase approach rate (DT and HR) or require heavy memorization (Flashlight).
 
-In [Catch the Beat](Catch_the_Beat "wikilink") mode, Hidden causes notes to fade out about halfway down the screen. This is the most commonly used mod in Catch the Beat, since the other difficulty increasing mods are very difficult. They either greatly increase approach rate (DT and HR) or require heavy memorization (Flashlight).
+In [osu!mania](osu!/wiki/Game_Modes/osu!mania) mode, Hidden can be used by pressing Fade In mod again. It is an inverse of Fade In where the bottom of the conveyor is blocked instead. Your judgement bar will be invisible too, but not the keys.
 
-In [osu!mania](osu!mania "wikilink") mode, Hidden can be used by pressing Fade In mod again. It is an inverse of Fade In where the bottom of the conveyor is blocked instead. Your Judgement bar will be invisible too, but not the keys.
+#### Fade In
 
-#### <img src="FadeIn.png" title="fig:FadeIn.png" alt="FadeIn.png" width="69" height="65" /> Fade In
+![Fade In mod icon](/shared/FadeIn.png "Fade In mod icon")
 
-***Score multiplier: 1.00*** (osu!mania only)
+**Score multiplier**: 1.00 (osu!mania only)
 
 This mod is specially for osu!mania only. It causes notes to fade in as they approach the judgement bar. The speed at which they fade is based on your combo, with higher combos causing the notes to fade in faster and remain invisible for longer.
 
 Clicking Fade In again will turn on Hidden mod.
 
 For a comparison, see pictures below.
 
-<File:GM> o!m FI.jpg|Fade In <File:GM> o!m HD.jpg|Hidden <File:GM> o!m FI(2).jpg|Hidden (DDR style) <File:GM> o!m HD(2).jpg|Fade In (DDR style) <File:Flashlight> Mania.jpg|Flashlight
+![Fade In](img/GM_o!m_FI.jpg "Fade In") ![Hidden](img/GM_o!m_HD.jpg "Hidden") ![Hidden (DDR style)](img/GM_o!m_FI2.jpg "Hidden (DDR style)") ![Fade In (DDR style)](img/GM_o!m_HD2.jpg "Fade In (DDR style)") ![Flashlight](img/Flashlight_Mania.jpg "Flashlight")
```

Comment:
Shouldn't Flashlight_Mania be in the Flashlight section, rather than the Fade In section?